memebot/httphandler/handler.go

```diff
@@ -39,7 +39,7 @@ func (h *Handler) handleEcho(cmd string, msg chat1.MsgSummary) error {
 			returnedMemeList := meme.ReturnMemeList()
 			var listMemes string
 			for k, _ := range returnedMemeList {
-				listMemes = fmt.Sprintf("%s \n %s", listMemes, k)
+				listMemes = fmt.Sprintf("%s \n%s", listMemes, k)
```

Comment:
```suggestion
				listMemes = fmt.Sprintf("%s\n%s", listMemes, k)
```
Might as well get rid of that space too unless you want the trailing space on that line for some reason?